components/company/Hero.vue

```diff
@@ -26,7 +26,7 @@
 
 <template>
   <div>
-    <VSheet :width="xs ? '90vw' : '65vw'">
+    <VCard :width="xs ? '90vw' : '65vw'" md="50v" variant="flat">

```

Comment:
I think we should set a max height for every viewport. When the height is the same as this defined limit, for any given viewport, the description should be truncated with `nowrap` and `ellipsis`